.buildkite/pipeline.yml

```diff
@@ -8,3 +8,19 @@ steps:
       - nix --extra-experimental-features "nix-command flakes" build .#packages.aarch64-darwin.nix-installer -L
       - cp result/bin/nix-installer ./nix-installer-aarch64-darwin
       - buildkite-agent artifact upload nix-installer-aarch64-darwin
+  - label: nix-installer-x86_64-linux
+    agents:
+      system: x86_64-linux
+      nix: 1
+    command:
+      - nix --extra-experimental-features "nix-command flakes" build .#packages.x86_64-linux.nix-installer-static -L
```

Comment:
We have a `x86_64 variants` github actions job, it would be nice to drop that and include those steps here, since this machine is a lot bigger.

Reply:
Shouldn't be too difficult (on both counts). I'll create tickets.